capellambse_context_diagrams/context.py

```diff
@@ -124,13 +125,6 @@ def __get__(  # type: ignore
 class ClassTreeAccessor(ContextAccessor):
     """Provides access to the tree view diagrams."""
 
-    # pylint: disable=super-init-not-called
-    def __init__(
-        self, diagclass: str, render_params: dict[str, t.Any] | None = None
-    ) -> None:
-        self._dgcls = diagclass
-        self._default_render_params = render_params or {}
```

Comment:
Did you test that removal is ok? The only difference here is that `super().__init__` from capellambse isn't called. Maybe there was a purpose for this? I don't know anymore.